packages/flutter/test/cupertino/theme_test.dart

```diff
@@ -123,4 +135,24 @@ void main() {
       expect(theme.scaffoldBackgroundColor, CupertinoColors.black);
     },
   );
+
+  testWidgets('Theme has default IconThemeData, which is derived from the theme\'s primary color', (WidgetTester tester) async {
```

Comment:
nit, don't escape stuff. If there's single quotes in the text, just use double quotes for the string literal.